frontend/src/assets/img/sprites.svg

```diff
@@ -0,0 +1,595 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="display:none;">
```

Comment:
Same with as the other uswds comment above.

Reply:
@lauraGgit same as above https://github.com/nciinc/fs-intake-module/issues/125